src/scanpy/plotting/_tools/paga.py

```diff
@@ -1331,7 +1331,7 @@ def moving_average(a):
                 cmap=color_map_anno,
             )
             if show_yticks:
-                anno_axis.set_yticklabels(["", anno, ""], fontsize=ytick_fontsize)
+                anno_axis.set_yticks(range(3), [anno, "", ""], fontsize=ytick_fontsize)
```

Comment:
I don’t think this has ever been visible, so it’s unclear if this fix is right.

(the other fix in this file *is* visually correct)